### PR TITLE
web: Add link preview meta tags for pay routes

### PR DESCRIPTION
**Pull request by backspace**
This is a step toward payment request links showing appropriate
previews when shared on social media.

I tried getting Fastboot working to actually expose these for link previews and failed… but it seems worth having them here anyway?

### CODE REVIEW

File: packages/web-client/app/components/card-pay/merchant-payment-request-header/index.hbs [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<InHead>
+  {{!-- template-lint-disable no-forbidden-elements --}}
+  <meta property='og:image' content={{this.cardPayLogo}} />
```
Comment (Aierie) [inline packages/web-client/app/components/card-pay/merchant-payment-request-header/index.hbs:3]: OG protocol doesn't support SVG, I will change this to a PNG.

Comment (Aierie) [reply packages/web-client/app/components/card-pay/merchant-payment-request-header/index.hbs:3]: Also needs to be an absolute URL.

**Review by Aierie — COMMENTED** [review]
Thanks for starting this, it should make payment sharing more user-friendly.

File: packages/web-client/app/components/card-pay/merchant-payment-request-header/index.hbs (hunk 1 above)
Comment (backspace) [reply packages/web-client/app/components/card-pay/merchant-payment-request-header/index.hbs]: Thanks for these fixes!

**Pull request merged**